vispy/util/fetching.py

```diff
@@ -234,9 +235,10 @@ def _fetch_file(url, file_name, print_destination=True):
     initial_size = 0
     # Checking file size and displaying it alongside the download url
     n_try = 3
+    contxt = ssl._create_unverified_context()
     for ii in range(n_try):
         try:
-            data = urllib.request.urlopen(url, timeout=15.)
+            data = urllib.request.urlopen(url, timeout=15., context=contxt)
```

Comment:
Sorry, but please remove this. Someone else has tried this in the past. This is not the right solution and completely negates the purpose of an HTTPS URL. I should go find that old conversation about this.

Reply:
I am sorry. I was just trying to see if the SSL check was the only thing causing this.
I found the issue [https://stackoverflow.com/questions/42098126/mac-osx-python-ssl-sslerror-ssl-certificate-verify-failed-certificate-verify] 
I think mac OS doesn't come preinstalled with certificates.
This post gives a way to install it but I am not sure how to do that on azure